x-pack/plugins/security_solution/public/explore/components/risk_score/risk_score_onboarding/risk_score_doc_link.tsx

```diff
@@ -8,18 +8,25 @@
 import { EuiLink } from '@elastic/eui';
 import React from 'react';
 import { RiskScoreEntity } from '../../../../../common/search_strategy';
-import { RISKY_HOSTS_DOC_LINK, RISKY_USERS_DOC_LINK } from '../../../../../common/constants';
+import {
+  RISKY_HOSTS_DOC_LINK,
+  RISKY_USERS_DOC_LINK,
+  RISKY_ENTITY_SCORE_DOC_LINK,
+} from '../../../../../common/constants';
 import { LEARN_MORE } from '../../../../overview/components/entity_analytics/risk_score/translations';
 
 const RiskScoreDocLinkComponent = ({
   riskScoreEntity,
   title,
 }: {
-  riskScoreEntity: RiskScoreEntity;
+  riskScoreEntity?: RiskScoreEntity;
   title?: string | React.ReactNode;
 }) => {
-  const docLink =
-    riskScoreEntity === RiskScoreEntity.user ? RISKY_USERS_DOC_LINK : RISKY_HOSTS_DOC_LINK;
+  const docLink = ((scoreEntity) => {
+    if (!scoreEntity) return RISKY_ENTITY_SCORE_DOC_LINK;
+    if (scoreEntity === RiskScoreEntity.user) return RISKY_USERS_DOC_LINK;
+    return RISKY_HOSTS_DOC_LINK;
+  })(riskScoreEntity);
```

Comment:
Same with the `if` and the IIEF. 
We can extract the function outside the component, or wrap i in a `useMemo`:

```suggestion
  const docLink = useMemo(() => {
    if (!riskScoreEntity) {
      return RISKY_ENTITY_SCORE_DOC_LINK;
    }
    return riskScoreEntity === RiskScoreEntity.user ? RISKY_USERS_DOC_LINK : RISKY_HOSTS_DOC_LINK;
  }, [riskScoreEntity]);
```